Lib/sysconfig.py

```diff
@@ -112,6 +112,11 @@ def _safe_realpath(path):
 # PC/VS7.1
 if os.name == "nt" and "\\pc\\v" in _PROJECT_BASE[-10:].lower():
     _PROJECT_BASE = _safe_realpath(os.path.join(_PROJECT_BASE, pardir, pardir))
+# PC/VS9.0/amd64
+if (os.name == "nt" 
+   and os.path.basename(os.path.dirname(os.path.dirname(_PROJECT_BASE))).lower() == "pc" 
+   and os.path.basename(os.path.dirname(_PROJECT_BASE)).lower().startswith("vs")):
```

Comment:
Ah, moreover: we *do have* a buildbot using VS 2008. I'm now trying to fix all buildbots, so I had to fix issues on this VS 2008 buildbot too: http://bugs.python.org/issue30313

But I guess that it's ok to have a buildbot on a system not officially fully supported.